packages/teams-js/package.json

```diff
@@ -18,6 +18,12 @@
     "prettier": "prettier --write '**/*.{ts,js,css,html}'",
     "test": "jest"
   },
+  "dependencies": {
+    "debug": "4.3.3"
+  },
```

Comment:
IIRC, there was some trouble when we added the dependency on es6-promise here originally and so we had to do something different? Does that apply here?

I think maybe it means we have to document other libraries that developers have to reference if they pull TeamsJS from the CDN?

Does debug have a minimum supported browser version?

Reply:
afaik, that was not because of the dependency itself, but because we called into it through `require('es6-promise')` which doesn't work. 

This bundles `debug` into our bundle (the one we deploy to CDN) and there's no need for the developer to reference anything extra for it to work.

The documentation does not specify their browser support matrix (not surprising, tbh). It is used widely across a lot of projects, including things like Fluid at Microsoft, so I would be very surprised if we had to support a browser it doesn't. Also, the project itself isn't very complicated and, from my skim through the code, does not use any modern potentially limiting browser features.